core/foundation/CMakeLists.txt

```diff
@@ -44,7 +44,7 @@ if(root7)
   ROOT_ADD_TEST_SUBDIRECTORY(v7/test)
 endif()
 
-ROOT_OBJECT_LIBRARY(Foundation ${FOUNDATION_SOURCES})
+ROOT_LINKER_LIBRARY(Foundation ${FOUNDATION_SOURCES})
```

Comment:
Does this means that in "all" configuration we now have 2 new libraries (libFoundation and libROOTLite)? @Axel-Naumann is that the intent?

Reply:
Yup! :+1: the idea was that actually in case of users will be able easier to use MiniRNTuple outside of ROOT.